core/src/impl/Kokkos_TaskQueue_impl.hpp

```diff
@@ -381,7 +381,7 @@ KOKKOS_FUNCTION void TaskQueue<ExecSpace, MemorySpace>::schedule_runnable(
 
   // If we don't have a dependency, or if pushing onto the wait queue of that
   // dependency failed (since the only time that queue should be locked is when
-  // the task is transitioning to complete??!?)
+  // the task is transitioning to complete?)
```

Comment:
Drop this change

Reply:
From Christian
```
core/src/impl/Kokkos_TaskQueue_impl.hpp", line 384: warning: support for trigraphs is disabled
    // the task is transitioning to complete??!?)

Reply:
this only happens with the fujitsu compiler so is related to adding fujistu compiler support